src/Spooler.php

```diff
@@ -1,25 +1,53 @@
 <?php
 
+/*
+ * (c) Henrik Bjornskov <henrik@bjrnskov.dk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Stampie;
 
 use Stampie\Spool\Storage;
 
+/**
+ * @package Stampie
+ */
 final class Spooler implements Mailer
 {
+    /**
+     * @var Mailer
+     */
     private $mailer;
+
+    /**
+     * @var Storage
+     */
     private $storage;
 
+    /**
+     * @param Mailer  $mailer
+     * @param Storage $storage
+     */
     public function __construct(Mailer $mailer, Storage $storage)
     {
-        $this->mailer = $mailer;
+        $this->mailer  = $mailer;
         $this->storage = $storage;
     }
 
+    /**
+     * @param Recipient $to
+     * @param Message   $message
+     */
```

Comment:
there is no value at all in these phpdocs as they just duplicate the info of the method signatures